.clang-tidy

```diff
@@ -7,6 +7,7 @@ Checks:
   -modernize-avoid-c-arrays,
   -modernize-use-trailing-return-type,
   -modernize-macro-to-enum,
+  -modernize-use-designated-initializers,
```

Comment:
Otherwise clang-tidy complains about not using designated initializers in VersionFunctions.h.

Not clear why since we specify c++17 in the clang-tidy config file.